backend/backend-schema-proposal.md

```diff
@@ -39,59 +39,128 @@ GET `/request/{van_id}`
 GET `/location`
 
 - Subscribes the client to SSE triggered by new coordinates to any van
-- Responses are in the format `(lat, lon, route_id, van_id, timestamp)` where
-  `van_id` & `route_id` are 8-bit ints, `lat` & `lon` are 64-bit doubles, and
-  `timestamp` is a UNIX timestamp
 - Any response with an earlier timestamp than the most recent should be discarded
 - Any timestamp more than 1 min in the future should be discarded
```

Comment:
maybe add that it will show an alert on the admin dashboard that there is a significant amount of clock drift

Reply:
I don't fully get what you mean.